.github/workflows/release.yml

```diff
@@ -53,22 +67,68 @@ jobs:
         with:
           path: ./artifacts
 
-      - name: Create archives
+      - name: Prepare release assets
         run: |
+          mkdir -p release
           cd artifacts
```

Comment:
As you can see on the previous job's step, there is already an artifact folder I create to handle releases assets files, maybe there is no need to create the release folder ?